homeassistant/components/modbus/modbus.py

```diff
@@ -195,8 +196,8 @@ def __init__(self, hass, client_config):
             },
         }
 
-    def _log_error(self, exception_error: ModbusException, error_state=True):
-        log_text = "Pymodbus: " + str(exception_error)
+    def _log_error(self, text: str, error_state=True):
+        log_text = "Pymodbus: " + text
```

Comment:
```suggestion
        log_text = f"Pymodbus: {text}"
```

Reply:
I just managed to unwillingly offend frenck for which I apologize  (he is right, I should have read twice what I wrote), but it  shows that my assesment yesterday was correct.

I might another day correct your suggestion, but right now I need to take a break before I damage more.

Thanks for your help.

Reply:
This change is minimal and correct. Push of the button. Thanks @MartinHjelmare 👍